etc/config/cuda.amazon.properties

```diff
@@ -42,7 +42,6 @@ compiler.nvcc11u1.semver=11.0.3
 compiler.nvcc111.exe=/opt/compiler-explorer/cuda/11.1.0/bin/nvcc
 compiler.nvcc111.options=--compiler-bindir /opt/compiler-explorer/gcc-10.2.0/bin
 compiler.nvcc111.semver=11.1.0
-compiler.nvcc111.alias=nvc111
```

Comment:
removing this alias might break some old URLs that referred to the old version, but I see this was missing from the list anyway...so not a problem I think!